build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+    ext {
+        projectGroupName 	= 'com.epam.repair'
+        projectVersion 		= '1.0-SNAPSHOT'
+        mysqlConnectorJavaVersion = '8.0.18'
+        springBootVersion 	= '2.2.0.RELEASE'
+        postgreSqlVersion = '42.2.8'
+        junitVersion 		= '4.12'
+        restAssuredVersion 	= '2.9.0'
+    }
+
+    repositories {
+        mavenCentral()
+        maven { url "https://repo.spring.io/snapshot" }
+        maven { url "https://repo.spring.io/milestone" }
+    }
+
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.2.0.RELEASE")
+    }
+}
+
+allprojects {
+    group projectGroupName
+    version projectVersion
+}
+
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'maven'
+    apply plugin: 'idea'
+    apply plugin: 'jacoco'
+    apply plugin: 'io.spring.dependency-management'
+
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
```

Comment:
Java 11?

Reply:
- failed to read class file /home/siarhei_arkhutsik/IdeaProjects/repair-service_gradle/dao-api/build/classes/java/test/com/epam/repair/dao/TestConfig.class
- Unsupported class file major version 55

with java 11.

Reply:
... and what?